### PR TITLE
Add a cookbook for Normandy events

### PR DESCRIPTION
**Pull request by tdsmith**
Describe how to do some common counting operations on the Normandy events in the events table.

### CODE REVIEW

**Comment by tdsmith** [comment]
@sunahsuh, you suggested something cookbook-like might be helpful in #181; is this a good start?

File: src/cookbooks/normandy_events.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+To fetch a count of events by reason and branch in PySpark:
+
+```python
+unenrollments_by_reason = (
+  events
+  .filter(events.event_category == "normandy")
+  .filter(events.event_method == "unenroll")
+  .filter(events.event_string_value == EXPERIMENT_SLUG)
+  .filter(events.submission_date_s3 > "20181000")
```
Comment (robhudson) [inline src/cookbooks/normandy_events.md]: Did you want to use the `EXPERIMENT_START` constant here too?

Comment (tdsmith) [reply src/cookbooks/normandy_events.md]: I did! Thanks for catching that.

**Comment by tdsmith** [comment]
I got some feedback that this was useful so I'm going to go ahead and land it; Sunah, please still feel free to revert or comment!

**Pull request merged**